Gems/Terrain/Code/Source/TerrainRenderer/TerrainMeshManager.cpp

```diff
@@ -648,12 +719,23 @@ namespace Terrain
     template<typename Callback>
     void TerrainMeshManager::ForOverlappingSectors(const AZ::Aabb& bounds, Callback callback)
     {
+        const AZ::Vector2 boundsMin2d = AZ::Vector2(bounds.GetMin().GetX(), bounds.GetMin().GetY());
+        const AZ::Vector2 boundsMax2d = AZ::Vector2(bounds.GetMax().GetX(), bounds.GetMax().GetY());
```

Comment:
Nit: You can just do boundsMin2d(bounds.GetMin()) and boundsMax2d(bounds.GetMax())